### PR TITLE
chore: update abdussalam02

### PR DESCRIPTION
**Pull request by abdussalam02**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1015"></a>

### CODE REVIEW

**Review by nathgoutam93 — APPROVED** [review]

**Review by krshdev — APPROVED** [review]
This looks nice to me, but as said above, just resolve conflicts and you are good to go! :D

**Comment by abdussalam02** [comment]
> Resolve the branch conflict otherwise looks good to me
Could you explain to me how to do it?

**Review by abdussalam02 — COMMENTED** [review]
It's all good now you can merge it.

**Review by krshdev — APPROVED** [review]

**Review by emmalearnscode — APPROVED** [review]

**Comment by emmalearnscode** [comment]
There is still a merge conflict that needs fixing. If you go to your repo, do fetch upstream, then in your branch do git pull. The merge conflict should show what the problem is and you can edit the file as you want it then make a new commit and push.

**Review by Pradumnasaraf — APPROVED** [review]
Everything LGTM only needs to resolve the merge conflict, you can refer to [official docs ](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line).
If you need any help to ping us as `Emma` said above :)

**Review by Jassi10000 — CHANGES_REQUESTED** [review]
@abdussalam02 Hey buddy !! pls resolve the remaining merge conflicts

**Review by KendallDoesCoding — APPROVED** [review]
This seems good, only the conflicts need to be resolved now.

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
LGTM!, please resolve the merge conflicts :), so that PR can get merged smoothly.
Thank You.

**Review by GaryStriving — APPROVED** [review]

**Review by krshdev — APPROVED** [review]

**Review by schmelto — APPROVED** [review]

**Comment by schmelto** [comment]
Could you resolve the merge conflicts? @abdussalam02

**Review by Panquesito7 — APPROVED** [review]

**Review by krshdev — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by krshdev — APPROVED** [review]

File: public/data/abdussalam_02.json [hunk 1]
```diff
@@ -5,22 +5,22 @@
...
     },
     {
-      "name": "Take a look at my GitHub",
+      "name": "Take a peek at my work",
       "url": "https://github.com/abdussalam02",
       "icon": "github"
     },
     {
-      "name": "I'm on Telegram most of the time",
+      "name": "Get in touch with me",
```
Comment (KendallDoesCoding) [inline public/data/abdussalam_02.json:18]: Will be better if its, get in touch with me on telegram? What do you think?

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

File: public/data/abdussalam_02.json (hunk 1 above)
Comment (abdussalam02) [reply public/data/abdussalam_02.json:18]: sure

Comment (KendallDoesCoding) [reply public/data/abdussalam_02.json]: thanks

**Comment by KendallDoesCoding** [comment]
> ## Fixes Issue
> ## Changes proposed
> ## Check List (Check all the applicable boxes)
> * [x]  My code follows the code style of this project.
> * [ ]  My change requires changes to the documentation.
> * [ ]  I have updated the documentation accordingly.
> * [x]  All new and existing tests passed.
> * [ ]  This PR does not contain plagiarized content.
> * [ ]  The title of my pull request is a short description of the requested changes.
> 
> ## Screenshots
> ## Note to reviewers
> [](https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1015)

It says you made some edits in README.md, what did you do there?

**Review by KendallDoesCoding — APPROVED** [review]

**Pull request merged**